sem_server_ros/Triple new_triple
---
std_msgs/String feedback
